python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -138,9 +138,12 @@ def fit(self, data, epochs=1, batch_size=32):
 
             # data transformation
             if isinstance(data, tuple):
+                if batch_size % self.num_processes != 0:
+                    warnings.warn("'batch_size' cannot be divided with no remainder by "
+                                  "'self.num_processes'.")
```

Comment:
maybe we can should them the exact value of batch_size and self.num_processes

Reply:
oh basically, we only want the warning message contains the rich output.
warnings.warn(f'......, we got batch_size = {batch_size} and "num of processes" = {self.num_processes}')